parallel-consumer-python/pyallel_consumer/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.confluent.parallelconsumer</groupId>
+    <artifactId>python</artifactId>
+    <version>0.1.0a0</version>
```

Comment:
alpha 0?

Reply:
Yeh I tried doing `SNAPSHOT` but PyPi didn't like it
I then tried just `a` for alpha but the packaging tools automatically added a 0 to the end if I didn't give one
So now explicitly going with `a0` which yes stands for alpha 0